test/lunatech/lunchplanner/controllers/RestControllerSpec.scala

```diff
@@ -69,12 +61,17 @@ class RestControllerSpec extends ControllerSpec with MockFactory {
   }
   private val cookieBaker: MockedCookieBaker = mock[MockedCookieBaker]
 
+  class MockedAuthenticate
+      extends Authenticate(configuration: Configuration, mock[WSClient]) {}
```

Comment:
Can't you mock the configuration too somehow? 🤔

Reply:
Just wondering, looks good otherwise

Reply:
I tried for a bit and ended up with something similar to the following snippet but couldn't get it to run. Maybe there's something about scalamock I am missing

`val mockedConf = mock[Configuration]
  (mockedConf.get[String] _)
    .expects(*)
    .returns("")
    .anyNumberOfTimes()`

Reply:
hmm Im not so sure either, wouldn't waste too much time on it and just keep it as is.